scripts/ci/_utils.sh

```diff
@@ -1419,6 +1419,7 @@ function prepare_prod_build() {
                 "--build-arg" "CONSTRAINT_REQUIREMENTS=https://raw.githubusercontent.com/apache/airflow/v1-10-test/requirements/requirements-python${PYTHON_MAJOR_MINOR_VERSION}.txt"
                 # TODO We should uncomment it just before we merge this change to master/v1_10_test
```

Comment:
Also what about this `TODO`?

Reply:
This is already corrected in #8234. Also we should adapt it slightly after we release 1.10.10 (for releases < 1.10.10 it should be pointing to 1.10.10 and for >= 1.10.10 it should point to ${AIRFLOW_VERSION}